silx/math/fft/fftw.py

```diff
@@ -104,13 +108,6 @@ def _allocate(self, shape, dtype):
 
 
     def check_array(self, array, shape, dtype, copy=True):
-        """
```

Comment:
did you voluntarily remove the documentation of the function ?

Reply:
The documentation was incorrect - the function does not return a new array. So I preferred to remove it.